settings.gradle.kts

```diff
@@ -24,5 +24,3 @@ plugins {
   // Automatic toolchain provisioning
   id("org.gradle.toolchains.foojay-resolver-convention") version "0.4.0"
 }
-
-rootProject.name = "gtnhgradle"
```

Comment:
Changing this will change the maven publishing coordinates for the plugin, so make sure to change it back in build.gradle.kts's maven configs. I'd also rather keep it and rename the project to GTNHGradle here, assigning the name explicitly avoids another intellij import bug for me, where it sometimes decides to import a project as `com.gtnewhorizons.gtnhgradle` instead of `gtnhgradle`.